README.md

```diff
@@ -20,38 +20,6 @@ The intention of this project is to create the easily configurable template, sum
 git clone https://github.com/Praqma/JenkinsAsCodeReference.git
 ```
 
-* Set proxy variables. You have to do it even if you are not using a proxy because there is [a bug in docker-compose](https://github.com/docker/compose/issues/3281) which makes args return None instead of empty string. Because of that, if you don't use proxy, you have to define empty environment variables http_proxy, https_proxy, no_proxy. Otherwise you would have those variables set to point out your proxy settings
-
-```
-export http_proxy=<empty or proxy address>
-export https_proxy=<empty or proxy address>
-export no_proxy=<empty or proxy address>
-export JAVA_OPTS=<empty or -Dhttps.proxyHost=<proxy address> -Dhttps.proxyPort=<proxy port> -Dhttp.nonProxyHosts=localhost,127.0.0.1,*.whatever.com -Dhttp.proxyHost=<proxy address> -Dhttp.proxyPort=<proxy port>
-```
-
-or
-
-```
-cat > ~/.bashrc <<- EOM
-export http_proxy=<empty or proxy address>
-export https_proxy=<empty or proxy address>
-export no_proxy=<empty or proxy address>
-export JAVA_OPTS=<empty or -Dhttps.proxyHost=<proxy address> -Dhttps.proxyPort=<proxy port> -Dhttp.nonProxyHosts=localhost,127.0.0.1,*.whatever.com -Dhttp.proxyHost=<proxy address> -Dhttp.proxyPort=<proxy port>
-EOM
-source ~/.bashrc
-```
-
-* Create backup directories - they will be used to store build history, user content, Gradle cache and Docker images from the local registry. Also we will use jenkins-backup/workspace directory for mapping to /root/workspace inside slave docker container - it needs to be done for working docker pipeline plugin properly. See [JENKINS-35217](https://issues.jenkins-ci.org/browse/JENKINS-35217) for details. You can find the list of all volumes used by this setup inside [dockerizeit/docker-compose.yml](dockerizeit/docker-compose.yml)
```

Comment:
The same here. If it works differently with docker-compose 1.7.0 then we should communicate that